### ISSUE TITLE
[BUG] MWS - tests fail on windows 11 - branch multi-wiki-support

### ISSUE THREAD

**Issue by pmario** [TiddlyWiki/TiddlyWiki5#8623]
log Windows 11

```
PS E:\git\tiddly\tiddlywiki\TiddlyWiki5> npm test

> tiddlywiki@5.3.6-prerelease test
> npm run build:test-edition && npm run test:multiwikiserver-edition


> tiddlywiki@5.3.6-prerelease build:test-edition
> node ./tiddlywiki.js ./editions/test --verbose --version --build index

Boot log:
  Startup task: load-modules
  Startup task: info after: load-modules before: startup
  Startup task: plugins after: load-modules before: startup
  Startup task: startup after: load-modules
  Startup task: multiwikiserver platforms: node before: story
  Startup task: story after: startup
  Startup task: commands platforms: node after: story
Executing command: version
5.3.6-prerelease
Executing command: build index
Executing command: rendertiddler $:/core/save/all test.html text/plain
Executing command: test
Randomized with seed 95951
Started
................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................File ./editions/test/test-store/large-file.txt generated with size 10485760 bytes
...File ./editions/test/test-store/large-file-0.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-1.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-2.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-3.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-4.txt generated with size 10485760 bytes
.File ./editions/test/test-store/large-file-0.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-1.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-2.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-3.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-4.txt generated with size 10485760 bytes
..(node:1716) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)
...............................................................................................................................................................................................................................................................................................................................................**.................................................................................................................................................................................................................................................................................................................................................


Suite error: AttachmentStore
  Message:
    Error: ENOTEMPTY: directory not empty, rmdir 'E:\git\tiddly\tiddlywiki\TiddlyWiki5\editions\test\test-store\files\3d919618de9ad6357fc92525c844f907b73f245e1345322276ada9fd68abbc60'
  Stack:
    undefinedError: ENOTEMPTY: directory not empty, rmdir 'E:\git\tiddly\tiddlywiki\TiddlyWiki5\editions\test\test-store\files\3d919618de9ad6357fc92525c844f907b73f245e1345322276ada9fd68abbc60'
        at Object.rmdirSync (node:fs:1215:11)
        at _rmdirSync (node:internal/fs/rimraf:260:21)
        at rimrafSync (node:internal/fs/rimraf:193:7)
        at node:internal/fs/rimraf:253:9
        at <Jasmine>
        at _rmdirSync (node:internal/fs/rimraf:250:7)
        at rimrafSync (node:internal/fs/rimraf:193:7)
        at Object.rmdirSync (node:fs:1209:14)
        at $:/plugins/tiddlywiki/multiwikiserver/modules/tests/test-attachment.js:66:16
        at <Jasmine>
Pending:

1) Tags Operator tests With all indexers should apply tags ordering in order of creation. TW v5.3.7+
  Temporarily disabled with xit


2) Tags Operator tests With no indexers should apply tags ordering in order of creation. TW v5.3.7+
  Temporarily disabled with xit

1352 specs, 1 failure, 2 pending specs
Finished in 124.518 seconds
Randomized with seed 95951 (jasmine --random=true --seed=95951)
Error: Tests failed with code 1
PS E:\git\tiddly\tiddlywiki\TiddlyWiki5>

```

**Comment by Jermolene** [comment]
Hi @pmario are you still seeing this error? The tests are running in CI and on my machine but I can investigate further if it is still an issue.

**Comment by pmario** [comment]
There still seems to be a problem with an "non empty" directory.

I did use: **Node.js version v20.14.0** for this test
Windows 11 pro
Powershell: PSVersion 5.1.22621.4249
in Windows Terminal: Version: 1.20.11781.0

```
PS E:\git\tiddly\tiddlywiki\TiddlyWiki5> npm run
Lifecycle scripts included in tiddlywiki@5.3.6-prerelease:
  start
    node ./tiddlywiki.js ./editions/multiwikiserver --mws-load-plugin-bags --build load-mws-demo-data --mws-listen
  test
    npm run build:test-edition && npm run test:multiwikiserver-edition

available via `npm run-script`:
  build:test-edition
    node ./tiddlywiki.js ./editions/test --verbose --version --build index
  test:multiwikiserver-edition
    node ./tiddlywiki.js ./editions/multiwikiserver/ --build load-mws-demo-data --mws-listen --mws-test-server http://127.0.0.1:8080/ --quit
  lint:fix
    eslint . --fix
  lint
    eslint .

PS E:\git\tiddly\tiddlywiki\TiddlyWiki5> npm test

> tiddlywiki@5.3.6-prerelease test
> npm run build:test-edition && npm run test:multiwikiserver-edition


> tiddlywiki@5.3.6-prerelease build:test-edition
> node ./tiddlywiki.js ./editions/test --verbose --version --build index

Boot log:
  Startup task: load-modules
  Startup task: info after: load-modules before: startup
  Startup task: plugins after: load-modules before: startup
  Startup task: startup after: load-modules
  Startup task: multiwikiserver platforms: node before: story
  Startup task: story after: startup
  Startup task: commands platforms: node after: story
Executing command: version
5.3.6-prerelease
Executing command: build index
Executing command: rendertiddler $:/core/save/all test.html text/plain
Executing command: test
Randomized with seed 04291
Started
........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................File ./editions/test/test-store/large-file.txt generated with size 10485760 bytes
....File ./editions/test/test-store/large-file-0.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-1.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-2.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-3.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-4.txt generated with size 10485760 bytes
.File ./editions/test/test-store/large-file-0.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-1.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-2.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-3.txt generated with size 10485760 bytes
File ./editions/test/test-store/large-file-4.txt generated with size 10485760 bytes
...(node:14996) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)
.*.*..............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................


Suite error: AttachmentStore
  Message:
    Error: ENOTEMPTY: directory not empty, rmdir 'E:\git\tiddly\tiddlywiki\TiddlyWiki5\editions\test\test-store\files\3d919618de9ad6357fc92525c844f907b73f245e1345322276ada9fd68abbc60'
  Stack:
    undefinedError: ENOTEMPTY: directory not empty, rmdir 'E:\git\tiddly\tiddlywiki\TiddlyWiki5\editions\test\test-store\files\3d919618de9ad6357fc92525c844f907b73f245e1345322276ada9fd68abbc60'
        at Object.rmdirSync (node:fs:1215:11)
        at _rmdirSync (node:internal/fs/rimraf:260:21)
        at rimrafSync (node:internal/fs/rimraf:193:7)
        at node:internal/fs/rimraf:253:9
        at <Jasmine>
        at _rmdirSync (node:internal/fs/rimraf:250:7)
        at rimrafSync (node:internal/fs/rimraf:193:7)
        at Object.rmdirSync (node:fs:1209:14)
        at $:/plugins/tiddlywiki/multiwikiserver/modules/tests/test-attachment.js:67:16
        at <Jasmine>
Pending:

1) Tags Operator tests With no indexers should apply tags ordering in order of creation. TW v5.3.7+
  Temporarily disabled with xit


2) Tags Operator tests With all indexers should apply tags ordering in order of creation. TW v5.3.7+
  Temporarily disabled with xit

1362 specs, 1 failure, 2 pending specs
Finished in 125.558 seconds
Randomized with seed 04291 (jasmine --random=true --seed=04291)
Error: Tests failed with code 1
PS E:\git\tiddly\tiddlywiki\TiddlyWiki5>
```

**Comment by pmario** [comment]
Close in favour of newer test runs
